### PR TITLE
Move Wiki to Docs

### PR DESCRIPTION
**Pull request by uilianries**
This PR allows the community to open new PRs, fixing or updating CCI docs.

I just copied the Wiki content to docs/<file>.md. As you can see, README now points to docs/ folder instead of Wiki page.

We should close Wiki after merging this PR, to avoid any possible duplication when reading the documentation.

Plus: I've added CONTRIBUTION.md, based on Conan project (maybe I should open a separated PR for that).

Related to #2101 

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+Contributing to Conan Center Index
```
Comment (danimtb) [inline .github/CONTRIBUTING.md:1]: The pipeline is not prepared to process this file. Why should it be under the .github folder?

**Comment by Croydon** [comment]
Organizational question: Would it be okay for the CCI team to leverage GitHub Actions to push the documentation to somewhere else (i.e. the GitHub wiki)?

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (Croydon) [reply .github/CONTRIBUTING.md:1]: This is a file with a special meaning. GitHub will provide a link to this explanation for contributors

Comment (danimtb) [reply .github/CONTRIBUTING.md:1]: yeah, now I remember. Should it be under *.github* or in the root folder? Maybe we can put it under *docs* as it is considered documentation as well https://docs.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors

**Comment by danimtb** [comment]
> Organizational question: Would it be okay for the CCI team to leverage GitHub Actions to push the documentation to somewhere else (i.e. the GitHub wiki)?

Currently, we are moving away from the wiki page. I think having the docs as markdown in the repo doesn't degrade the reader experience and improves the contributor one. We can consider pushing this documentation to somewhere else in the future, like the wiki (if requested) or even a proper section in [conan.io/center](https://conan.io/center) website.

**Comment by uilianries** [comment]
After reading Github docs, we can use `docs` too: https://docs.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (uilianries) [reply .github/CONTRIBUTING.md]: done!

File: README.md [hunk 2]
```diff
@@ -6,15 +6,20 @@ This repository includes a Continuous Integration system that will build automat
 [Pull Request](https://github.com/conan-io/conan-center-index/pulls).
 
 
-### You can learn how to contribute new recipes and how this build service works in [the wiki of the project](https://github.com/conan-io/conan-center-index/wiki).
+### Documentation
+
+* If you want to learn how to contribute new recipes, please read [docs/how_to_add_packages.md](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md).
+* Most common questions are listed in [docs/faqs.md](https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md).
+* The conan-center hook errors reported by CCI Bot can be found in the [docs/error_knowledge_base.md](https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md).
```
Comment (danimtb) [inline README.md:13]: I believe using relative links should work https://github.blog/2013-01-31-relative-links-in-markup-files/
```suggestion
* If you want to learn how to contribute new recipes, please read [docs/how_to_add_packages.md](docs/how_to_add_packages.md).
* Most common questions are listed in [docs/faqs.md](docs/faqs.md).
* The conan-center hook errors reported by CCI Bot can be found in the [docs/error_knowledge_base.md](docs/error_knowledge_base.md).
```

File: README.md [hunk 3]
```diff
@@ -6,15 +6,20 @@ This repository includes a Continuous Integration system that will build automat
...
+* The internal changes related to infrastructure can be checked in [docs/changelog.md](https://github.com/conan-io/conan-center-index/blob/master/docs/changelog.md).
```
Comment (danimtb) [inline README.md:14]:
```suggestion
* The internal changes related to infrastructure can be checked in [docs/changelog.md](docs/changelog.md).
```

File: docs/changelog.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+### 17-July-2020 - 18:54 CEST
```
Comment (danimtb) [inline docs/changelog.md:1]: please add the latest entries from the wiki (it was updated today)

File: docs/contributing.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+CCI follows the ["GitFlow"](https://datasift.github.io/gitflow/IntroducingGitFlow.html) branching model.
+Issues are triaged and categorized mainly by type (package request, bug...) and priority (high, medium...) using GitHub
+ labels.
+
+To contribute follow the next steps:
+
+1. Comment in the corresponding issue that you want to contribute the package/fix proposed. If there is no open issue, we strongly suggest
+   to open one to gather feedback.
+2. Check the [docs/how_to_add_packages.md](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) if are
```
Comment (danimtb) [inline docs/contributing.md:23]:
```suggestion
2. Check the [docs/how_to_add_packages.md](docs/how_to_add_packages.md) if are
```

File: README.md (hunk 2 above)
Comment (uilianries) [reply README.md:13]: that's great!

File: docs/changelog.md (hunk 4 above)
Comment (uilianries) [reply docs/changelog.md:5]: Done!

File: docs/changelog.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
+### 24-July-2020 - 12:52 CEST
+-  [fix] Renamed Jenkins project from conan-center-pull-request to cci to improve issues with long workspace paths in Windows agents.
+-  [feature] Updated Conan client to the 1.27.1 version in Windows and Mac agents.
```
Comment (danimtb) [inline docs/changelog.md:3]: Sorry, I forgot to update the time. It is important in order to track changes. I have updated the wiki as well

```suggestion
### 24-July-2020 - 13:05 CEST
- Renamed Jenkins project from `conan-center-pull-request` to `cci` to improve issues with long workspace paths in Windows agents.

### 24-July-2020 - 12:52 CEST
- Updated Conan client to the 1.27.1 version in Windows and Mac agents.
```

**Pull request merged**